workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/MapQualitySettings.cs

```diff
@@ -62,26 +73,27 @@ private float CheckoutDistanceInternal
         }
     }
 
-    private float checkoutDistanceCache = -1;
+    #region Editor functions
 
-    private void Awake()
+    private void Editor_ApplyCheckoutDistanceToTiles()
```

Comment:
If you add the attribute `[Conditional("UNITY_EDITOR")]` this function will automatically be compiled away if that symbol is not defined

Reply:
Ohh, neato

Reply:
What happens if this function is then called from outisde the editor? Does it ignore the call or does it cause an error?

Reply:
I'll use Jess's suggestion of `if (Application.isEditor)` for now, but this is a useful attribute to know about.